### ISSUE TITLE
False positive in Internet Explorer 10

### ISSUE THREAD

**Issue by Grawl** [ai/webp-in-css#11]
No errors, just `webp` class on `<body>`. But it does not support WebP.

<img width="447" alt="изображение" src="https://user-images.githubusercontent.com/846774/61681179-c7588680-ad4f-11e9-9995-7ded721099a2.png">

**Comment by Grawl** [comment]
as a workaround, I just remove `webp` from `<body>` and add `no-webp` to show images in IE 10 using [hack for IE detection](http://browserhacks.com/#hack-2f32c95ac8c021c463de0fdf685acb29):

```
var isInternetExplorer10 = (
	'behavior' in document.documentElement.style &&
	'-ms-user-select' in document.documentElement.style
);
if (isInternetExplorer10) {
	document.addEventListener('DOMContentLoaded', function() {
		document.body.classList.remove('webp');
		document.body.classList.add('no-webp');
	});
}
```

**Comment by ai** [comment]
Can you debug JS code? Where it goes wrong in IE10.

**Comment by ai** [comment]
@Grawl Can you test d8f6685? Just use `"webp-in-css": "ai/webp-in-css"` in dependencies?

**Comment by Grawl** [comment]
it works!

**Comment by ai** [comment]
Released in 0.1.2
